tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -369,8 +369,33 @@ private static void InitRemoteConfigurationManagement(Tracer tracer)
                     {
                         var rcmSettings = RemoteConfigurationSettings.FromDefaultSource();
                         var rcmApi = RemoteConfigurationApiFactory.Create(tracer.Settings.Exporter, rcmSettings, discoveryService);
-
-                        var configurationManager = RemoteConfigurationManager.Create(discoveryService, rcmApi, rcmSettings, serviceName, tracer.Settings.Environment, tracer.Settings.ServiceVersion);
+                        var tags = tracer.Settings.GlobalTags.Select(pair => pair.Key + ":" + pair.Value).ToList();
+
+                        var environment = tracer.Settings.Environment;
+                        if (!string.IsNullOrEmpty(environment))
+                        {
+                            tags.Add($"env:{environment}");
+                        }
+
+                        var serviceVersion = tracer.Settings.ServiceVersion;
+                        if (!string.IsNullOrEmpty(serviceVersion))
+                        {
+                            tags.Add($"version:{serviceVersion}");
+                        }
+
+                        var tracerVersion = rcmSettings.TracerVersion;
+                        if (!string.IsNullOrEmpty(tracerVersion))
+                        {
+                            tags.Add($"tracer_version:{tracerVersion}");
+                        }
+
+                        var hostName = PlatformHelpers.HostMetadata.Instance?.Hostname;
+                        if (!string.IsNullOrEmpty(hostName))
+                        {
+                            tags.Add($"host_name:{hostName}");
+                        }
```

Comment:
Consider extracting to a method?

Reply:
Done in 81be4953cd7e717c95bf3e50fd416c69dd26062a.